modules/gltf/gltf_document.cpp

```diff
@@ -30,6 +30,7 @@
 
 #include "gltf_document.h"
 
+#include "core/error/error_macros.h"
```

Comment:
Spot the potential issue :)
![Screenshot_20210809_083230](https://user-images.githubusercontent.com/4701338/128667778-dc66ad0c-8e97-4f31-9c88-2ff8fa35ca42.png)